api/tests/test_main.py

```diff
@@ -126,10 +126,10 @@ def setup_empty_files(setup_data_folder):
 def test_no_token_is_unauthorized():
     response = test_client.get("/tifs")
     response2 = test_client.post("/exact_zonal_stats")
-    assert response.status_code == 401
-    assert response2.status_code == 401
-    assert response.json() == {"detail": "Unauthorized"}
-    assert response2.json() == {"detail": "Unauthorized"}
+    assert response.status_code == 403
```

Comment:
I think this doesn't make sense. 403 HTTP Code stands for Forbidden resource, which is commonly related to authorisation rather than authentication.

Besides, [here](https://github.com/Vizzuality/amazonia-360/pull/57/files#diff-fa352d701f34c44e4c3166985bad5c428225da427f064fae143c85686bf1ce59R18) we are explicitly raising a 401 code with "Unauthorized" string as detail, but the assertion checks for a 403 code with "Not authenticated" message, which makes me think that either this code is not being reached or it's wrapped afterwards with some framework magic.

I'm all in for using framework provided magic rather than implementing custom stuff for tools that are built-in, but I think this case does not follow the web standards

Reply:
Yes it looks it is a know issue https://github.com/tiangolo/fastapi/issues/10177.